b2b-sample/guardio-insurance-sample-application/components/settingsComponents/application/application.js

```diff
@@ -0,0 +1,317 @@
+        }
+
+    };
+
+    const loadIdentityProviders = async () => {
+
+        const res = await listAllIdentityProviders({
+            session, offset: 0, limit: 20
+        });
+
+        if (res && res.identityProviders) {
+            setIdpList(res.identityProviders);
+        } else {
+            setIdpList([]);
+        }
+
+    };
+
+    const onOptionsAddClick = (step) => {
+        setActiveStep(step);
+    };
+
+    const onOptionsModalCloseClick = (step) => {
+        if (step) {
+            setOpenModal(false);
+            setActiveStep(undefined);
+        }
+    };
+
+    const onIdPSelected = async (idp) => {
+
+        const FIRST_AUTHENTICATOR = 0;
+        const details = await getDetailedIdentityProvider({id: idp.id, session});
+        const authenticator = details.federatedAuthenticators.authenticators[FIRST_AUTHENTICATOR];
+
+        /**
+         * This sample application will override the rest of
+         * steps which are added from other APIs/Dashboards.
+         * @type {*[]}
+         */
+        const authenticationSequence = {...application.authenticationSequence};
+
+        const existingProvidersInFirstStep = new Set(
+            authenticationSequence.steps[0].options
+                .map(({idp}) => idp)
+                .filter(name => name !== "LOCAL")
+        );
+
+        if (existingProvidersInFirstStep.has(idp.name)) {
+            alert("already exists!")
+            return;
+        }
+
+        const updatedSequence = Object.assign(authenticationSequence, {
+            steps: [
+                {
+                    id: 1,
+                    options: [
+                        {idp: 'LOCAL', authenticator: 'BasicAuthenticator'},
+                        {idp: idp.name, authenticator: authenticator.name}
+                    ],
+                }
+            ]
+        });
+
+        setApplication({
+            ...Object.assign(application, {
+                authenticationSequence: updatedSequence
+            })
+        });
+
+        setActiveStep(undefined);
+        setOpenModal(false);
+
+    };
+
+    const onIdPRemovedFromSequence = (step, option) => {
+
+        const authenticationSequence = {...application.authenticationSequence};
+        const updatedOptions = step.options.filter(o => o.idp !== option.idp);
+
+        const updatedSequence = Object.assign(authenticationSequence, {
+            steps: [
+                {
+                    id: 1,
+                    options: updatedOptions,
+                }
+            ]
+        });
+
+        setApplication({
+            ...Object.assign(application, {
+                authenticationSequence: updatedSequence
+            })
+        });
+
+    };
+
+    const onUpdateAuthenticationSeqClick = async () => {
+
+        const res = await patchApplication({
+            id: application.id,
+            partial: application,
+            session
+        });
+
+        console.log(res);
```

Comment:
Do we need this console log?

Reply:
This file is removed an updated with new set of files in a later PR,

### Files
- https://github.com/wso2/samples-is/blob/master/b2b-sample/apps/business-admin-app/pages/api/settings/application/getApplication/%5Bid%5D.ts
- https://github.com/wso2/samples-is/blob/master/b2b-sample/apps/business-admin-app/pages/api/settings/application/